src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -247,14 +248,52 @@ contract SecurityCouncilNomineeElectionGovernor is
         );
 
         ElectionInfo storage election = _elections[proposalId];
+        require(!election.isExcluded[account], "Nominee already excluded");
 
         election.isExcluded[account] = true;
         election.excludedNomineeCount++;
 
         emit NomineeExcluded(proposalId, account);
     }
 
-    /************** internal/private state mutating functions **************/
+    /// @notice Allows the nomineeVetter to explicitly include a nominee
+    /// @dev    Can be called only after a proposal has succeeded (voting has ended) and before the nominee vetting period has ended.
+    ///         Will revert if the provided account is already a nominee
+    function includeNominee(uint256 proposalId, address account) external onlyNomineeVetter {
```

Comment:
should we add a requirement so that the nomineeVetter doesn't add too many?

`require(compliantNomineeCount < targetNomineeCount)`

Reply:
Good catch! Added